docs/browser.md

```diff
@@ -41,16 +59,35 @@ Or if you are using `yarn`:
 yarn add @0x/mesh-browser
 ```
 
-We recommend using a tool like [Webpack](https://webpack.js.org/) to bundle the
-0x Mesh package and all your other code into a single JavaScript file.
+Similarly, the `@0x/mesh-browser-lite` NPM package can be installed by running:
+
+```
+npm install --save @0x/mesh-browser-lite
+```
+
+Or if you are using `yarn`:
+
+```
+yarn add @0x/mesh-browser-lite
+```
 
 ## Documentation
 
-Documentation for the `@0x/mesh-browser` package is available at
-[0x-org.gitbook.io/mesh/browser](https://0x-org.gitbook.io/mesh/browser).
+* Documentation for the `@0x/mesh-browser` package is available at
+[0x-org.gitbook.io/mesh/browser](https://0x-org.gitbook.io/mesh/browser-bindings/browser).
+* Documentation for the `@0x/mesh-browser-lite` package is available at
+[0x-org.gitbook.io/mesh/browser](https://0x-org.gitbook.io/mesh/browser-bindings/browser-lite).
+
+### Example usage
 
-## Example usage
+[The packages/webpack-example directory](../packages/webpack-example) includes a
+bare-bones example of how to use the `@0x/mesh-browser` and bundle everything
+together with Webpack.
 
-[The examples/browser directory](../examples/browser) includes a bare-bones
-example of how to use the `@0x/mesh-browser` and bundle everything together with
-Webpack.
+[The packages/webpack-example-lite directory](../packages/webpack-example)
+includes a bare-bones example of how to use the `@0x/mesh-browser-lite` and bundle
+everything together with Webpack. This is slightly different than the other example
+because it requires that WebAssembly bytecode be provided by the user of the package.
+WebAssembly binaries for each version of 0x-mesh that has an associated
+`@0x/mesh-browser-lite` package can be found in the
+[0x-mesh release notes](https://github.com/0xProject/0x-mesh/releases).
```

Comment:
Some of this is redundant and I think it's better to go into more detail above in the `@0x/mesh-browser-lite` section. Putting the information all in one place will make it easier for readers to understand.

```suggestion
everything together with Webpack.
```